modules/example-node/src/caching_cmm.ts

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ * this file except in compliance with the License. A copy of the License is
+ * located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { KmsKeyringNode, encrypt, decrypt, NodeCachingMaterialsManager, getLocalCryptographicMaterialsCache } from '@aws-crypto/client-node'
+
+export async function cachingCMMNodeSimpleTest () {
+  /* A KMS CMK is required to generate the data key.
+   * You need kms:GenerateDataKey permission on the CMK in generatorKeyId.
+   */
+  const generatorKeyId = 'arn:aws:kms:us-west-2:658956600833:alias/EncryptDecrypt'
+
+  /* Adding alternate KMS keys that can decrypt.
+   * Access to kms:Encrypt is required for every CMK in keyIds.
+   * You might list several keys in different AWS Regions.
+   * This allows you to decrypt the data in any of the represented Regions.
+   * In this example, I am using the same CMK.
+   * This is *only* to demonstrate how the CMK ARNs are configured.
+   */
+  const keyIds = ['arn:aws:kms:us-west-2:658956600833:key/b3537ef1-d8dc-4780-9f5a-55776cbb2f7f']
+
+  /* The KMS keyring must be configured with the desired CMKs
+   * In this case we are going to pass this keyring to the caching CMM,
+   * instead of using it directly.
+   */
+  const keyring = new KmsKeyringNode({ generatorKeyId, keyIds })
+
+  /* Create a cache to hold the material.
+   * In this case we use the local cache provided by the Encryption SDK.
```

Comment:
Please make all text in the node example match the corresponding edited text in the browser example.

Reply:
fixed